src/coins.cpp

```diff
@@ -532,7 +533,7 @@ void CCoinsViewCache::PopHistoryNode(uint32_t epochId) {
             // node.
```

Comment:
`tmpHistoryRoot` is only used in this case, so should probably be declared and initialized here.